mmv1/third_party/terraform/data_sources/data_source_secret_manager_secret.go

```diff
@@ -0,0 +1,23 @@
+package google
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceSecretManagerSecret() *schema.Resource {
+
+	dsSchema := datasourceSchemaFromResourceSchema(resourceSecretManagerSecret().Schema)
+	addRequiredFieldsToSchema(dsSchema, "secret_id")
+	addOptionalFieldsToSchema(dsSchema, "project")
+
+	return &schema.Resource{
+		Read:   dataSourceSecretManagerSecretRead,
+		Schema: dsSchema,
+	}
+}
+
+func dataSourceSecretManagerSecretRead(d *schema.ResourceData, meta interface{}) error {
+
```

Comment:
Had some time to look into this, and realized that this function doesn't set an ID for the datasource. This should address the test failure from before.

Reply:
This is fixed now.